marine/marine.py

```diff
@@ -228,7 +256,7 @@ def _add_or_get_filter(
         if filter_id < 0:
             err = None if err_msg_value is None else err_msg_value.decode("utf-8")
             if filter_id == c_int.in_dll(self._marine, "BAD_BPF_ERROR_CODE").value:
-                raise BadBPFException(err)
+                raise BadBPFException(f"Failed compiling the BPF: {err}")
```

Comment:
Why did you add extra text around this message? If its for clarity, why not to the other errors as well.

Reply:
This prefix used to be added by our c core, I moved it here to make the c errors more uniform